.github/workflows/latest-spring.yml

```diff
@@ -24,7 +24,7 @@ jobs:
     name: Java ${{ matrix.java }}
     strategy:
       matrix:
-        java: [8, 11]
+        java: [11]
```

Comment:
This starter works with Spring Boot 3, so I think this should be updated to include 17.